#!/usr/bin/env python
PACKAGE = "openh_rosbag_recoder"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("recording_switch", bool_t, 0, "Master switch to enable/disable rosbag recording feature", False)

exit(gen.generate(PACKAGE, "openh_rosbag_recoder", "RecorderParams"))